doc/source/how-to-run-flower-using-docker.rst

```diff
@@ -103,6 +103,144 @@ to the SuperLink with the ``--certificates`` flag.
+we copy the client-app code (``client.py`` and ``task.py``) into the image and set the entry
+point to ``flower-client-app``.
+
+Building the SuperNode Docker image
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next, we build the SuperNode Docker image by running the following command in the directory where
+Dockerfile and client-app code are located.
+
+.. code-block:: bash
+
+  $ docker build -t flwr_supernode:0.0.1 .
+
+We gave the image the name ``flwr_supernode``, and the tag ``0.0.1``. Remember that the here chosen
+values only serve as an example. You can change them to your needs.
+
+
+Running the SuperNode Docker image
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now that we have built the SuperNode image, we can finally run it.
+
+.. code-block:: bash
+
+  $ docker run --rm flwr_supernode:0.0.1 client:app \
+    --insecure \
+    --server 192.168.1.100:9092
+
+Let's break down each part of this command:
+
+* ``docker run``: This is the command to run a new Docker container.
+* ``--rm``: This option specifies that the container should be automatically removed when it stops.
+* | ``flwr_supernode:0.0.1``: The name the tag of the Docker image to use.
+* | ``client:app``: The object reference of the client-app (``<module>:<attribute>``).
+  | It points to the client-app that will be run inside the SuperNode container.
+* ``--insecure``: This option enables insecure communication.
+
+.. attention::
+
+  The ``--insecure`` flag enables insecure communication (using HTTP, not HTTPS) and should only be
+  used for testing purposes. We strongly recommend enabling
+  `SSL <https://flower.ai/docs/framework/how-to-run-flower-using-docker.html#enabling-ssl-for-secure-connections>`_
+  when deploying to a production environment.
+
+* | ``--server 192.168.1.100:9092``: This option specifies the address of the SuperLinks Fleet
+  | API to connect to. Remember to update it with your SuperLink IP.
+
+.. note::
+  Any argument that comes after the tag is passed to the Flower SuperNode binary.
+  To see all available flags that the SuperNode supports, run:
+
+  .. code-block:: bash
+
+    $ docker run --rm flwr/supernode:nightly --help
+
+Enabling SSL for secure connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To enable SSL, we will need to mount a PEM-encoded root certificate into your SuperNode container.
+
+Assuming the certificate already exists locally, we can use the flag ``-v`` to mount the local
+certificate into the container's ``/app/`` directory. This allows the SuperNode to access the
+certificate within the container. Use the ``--certificates`` flag when starting the container.
+
+.. code-block:: bash
+
+  $ docker run --rm -v ./ca.crt:/app/ca.crt flwr_supernode:0.0.1 client:app \
+    --insecure \
+    --server 192.168.1.100:9092 \
+    --certificates ca.crt
+
+Advanced Docker options
+-----------------------
+
 Using a different Flower version
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Shouldn't this and everything that follow be listed under SuperLink?

Reply:
the advanced options work with all of our Docker images